index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Jogo da Velha</title>
+<title>Tic-Tac-Toe</title>
```

Comment:
The page's title is a content read by the user. If you're planning to convert your whole interface to English, so that's fine keep your titles in English too. However, if you're planning to create a Portuguese UI, it's fine if you keep your texts in Portuguese too.